src/Emulator/Cores/RiscV/PLIC/PlatformLevelInterruptControllerBase.cs

```diff
@@ -20,10 +20,10 @@ namespace Antmicro.Renode.Peripherals.IRQControllers.PLIC
 {
     public abstract class PlatformLevelInterruptControllerBase : IPlatformLevelInterruptController, IDoubleWordPeripheral, IIRQController, INumberedGPIOOutput
     {
-        public PlatformLevelInterruptControllerBase(int numberOfSources, int numberOfContexts, bool prioritiesEnabled)
+        public PlatformLevelInterruptControllerBase(int numberOfSources, int numberOfContexts, bool prioritiesEnabled, uint extraInterrupts = 0)
```

Comment:
Changes in `PlatformLevelInterruptControllerBase` should go to a separate commit.

Reply:
Moved to a separate commit as requested.